grpclog/loggerv2.go

```diff
@@ -65,7 +67,11 @@ type LoggerV2 interface {
 // SetLoggerV2 sets logger that is used in grpc to a V2 logger.
 // Not mutex-protected, should be called before any gRPC functions.
 func SetLoggerV2(l LoggerV2) {
-	logger = l
+	grpclog.Logger = l
+	grpclog.DepthLogger = nil
+	if depthLogger, ok := l.(grpclog.DepthLoggerV2); ok {
+		grpclog.DepthLogger = depthLogger
+	}
```

Comment:
This can be simply:

`grpclog.DepthLogger, _ = l.(grpclog.DepthLoggerV2)`

Reply:
Fixed this.